lib/json_translate/translates/instance_methods.rb

```diff
@@ -1,12 +1,12 @@
 module JSONTranslate
   module Translates
     module InstanceMethods
-      def disable_fallback
-        toggle_fallback(false)
+      def disable_fallback(&block)
+        toggle_fallback(false, &block)
```

Comment:
could you use `yield if block_given?` instead please?

Reply:
Sure, I changed it. Admittedly, I'm not 100% sure what the difference is. Say, an exception during the `yield` I just added might not trigger the `ensure` block in the `toggle_fallback` method, right?

If you wish to enlighten me, I'd be happy to learn the reason. If not, that's also okay and you might just merge it ;)

Again, thank you for your time. I hope that you'll be able to get hold of the `hstore_translate` gem at some point.

Reply:
It is mainly for performance reasons. 
There is a performance penalty when using `&block` vs `yield` 😃